.github/workflows/eden_test.yml

```diff
@@ -1,10 +1,40 @@
 ---
-name: Test Eden WF from lf-edge/eden
-on:
-  workflow_dispatch:
+name: lf-edge/eden Test suite
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - "master"
+      - "[0-9]+.[0-9]+"
+      - "[0-9]+.[0-9]+-stable"
+    paths-ignore:
+      - 'docs/**'
+      - 'pkg/pillar/docs/**'
+  pull_request_review:
+    types: [submitted]
+    paths-ignore:
+      - 'docs/**'
+      - 'pkg/pillar/docs/**'
+
+concurrency:
+  group: eden_test_suite
```

Comment:
Doesn't this mean we can run a single workflow across all PRs etc?
The examples I see of concurrency uses a more specific name. I think we might need a combination of https://docs.github.com/en/actions/using-jobs/using-concurrency#example-only-cancel-in-progress-jobs-or-runs-for-the-current-workflow and https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-a-fallback-value

Reply:
@eriknordmark Yes you are correct, need to group the workflows better so that multiple workflows are triggered only associated with the 1 PR is cancelled. 

https://stackoverflow.com/questions/66335225/how-to-cancel-previous-runs-in-the-pr-when-you-push-new-commitsupdate-the-curre